src/frontend/src/utils/statusList/statuses/editInProgressStatus.ts

```diff
@@ -52,6 +53,12 @@ export const editInProgressStatus = () => {
       },
       CPO: {
         showSubmitButton: true,
+        Pages: {
+          review: {
+            accessControl: false,
+            showEditButton: true,
```

Comment:
This wont be needed because they wont see the page anyways because of accessControl being false